src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -603,6 +603,24 @@ export function makeDensityScatterPlotTooltip(
     };
 }
 
+const GENE_PANEL_PREFIX = 'gene_panel_';
+
+export function getGenePanelChartUniqueKey(profileId: string) {
+    return GENE_PANEL_PREFIX + profileId;
+}
+
+export function isGenePanelChart(uniqueKey: string) {
+    return uniqueKey.startsWith(GENE_PANEL_PREFIX);
+}
+
+export function chartUniqueKeyToProfile(meta: ChartMeta | string) {
```

Comment:
can we just have it accept string and then deference that string at point of call when necessary?